src/shared/utils/__tests__/utils.test.ts

```diff
@@ -75,6 +79,52 @@ describe("Util Functions", () => {
       },
       { version: SchemaId.v3 }
     );
+    wrappedV3DidDocument = await __unsafe__use__it__at__your__own__risks__wrapDocument(
+      {
+        "@context": [
+          "https://www.w3.org/2018/credentials/v1",
+          "https://www.w3.org/2018/credentials/examples/v1",
+          "https://schemata.openattestation.com/com/openattestation/1.0/OpenAttestation.v3.json",
```

Comment:
able to reduce noise by making use of spread operator to "edit" into variations? so it's clearer what attribute is different/needed in each case. eg:

```
  const sampleDocumentV3: v3.OpenAttestationDocument = {
    "@context": [
      "https://www.w3.org/2018/credentials/v1",
      "https://www.w3.org/2018/credentials/examples/v1",
      "https://schemata.openattestation.com/com/openattestation/1.0/OpenAttestation.v3.json",
    ],
    issuer: {
      name: "name",
      type: "OpenAttestationIssuer",
      id: "https://example.com",
    },
    issuanceDate: "2010-01-01T19:23:24Z",
    type: ["VerifiableCredential", "UniversityDegreeCredential", "OpenAttestationCredential"],
    credentialSubject: {
      id: "did:example:ebfeb1f712ebc6f1c276e12ec21",
      degree: {
        type: "BachelorDegree",
        name: "Bachelor of Science and Arts",
      },
    },
    openAttestationMetadata: {
      proof: {
        value: "0xabcf",
        type: v3.ProofType.OpenAttestationProofMethod,
        method: v3.Method.DocumentStore,
      },
      identityProof: {
        identifier: "whatever",
        type: v2.IdentityProofType.DNSTxt,
      },
      template: {
        url: "https://",
        name: "",
        type: v3.TemplateType.EmbeddedRenderer,
      },
    },
    name: "",
    reference: "",
    validFrom: "2010-01-01T19:23:24Z",
  };

  const sampleDocumentV3Did = {
    ...sampleDocumentV3,
    openAttestationMetadata: {
      ...sampleDocumentV3.openAttestationMetadata,
      proof: {
        ...sampleDocumentV3.openAttestationMetadata.proof,
        value: "did:ethr:0xabcf",
        method: v3.Method.Did,
        revocation: {
          type: v3.RevocationType.RevocationStore,
        },
      },
    },
  };

  const sampleDocumentV3TokenRegistry = {
    ...sampleDocumentV3,
    openAttestationMetadata: {
      ...sampleDocumentV3.openAttestationMetadata,
      proof: {
        ...sampleDocumentV3.openAttestationMetadata.proof,
        method: v3.Method.TokenRegistry,
      },
    },
  };
  ```